conf/openlibrary.yml

```
@@ -151,6 +151,10 @@ plugin_recaptcha:
     public_key: ''
     private_key: ''
 
+plugin_invisible_recaptcha:
+  public_key: ''
+  private_key: ''
```

```suggestion
    public_key: ''
    private_key: ''
```
Use consistent indentation amount in this file.